app/components/specific/cards/ClubActivitiesCard.js

```diff
@@ -19,7 +21,7 @@ export default class ClubActivitiesCard extends Component {
         title="ACTIVITIES"
         list={activities.map((activity, index) => ({
           key: index,
-          image: activity.athlete.profile,
+          image: { name: getIconName("runner"), color: theme.PrimaryColor },
```

Comment:
-_- that line is suspicious

Reply:
again, strava breaking change api! 
We have no longer access to club athlete details...